keep/api/models/db/migrations/versions/2025-03-07-17-49_aaec81b991bd.py

```diff
@@ -0,0 +1,307 @@
+        sa.Column("mac_address", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column("category", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column("manufacturer", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column("namespace", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column("is_manual", sa.Boolean(), nullable=True),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.func.current_timestamp(),
+            nullable=True,
+        ),
+        sa.ForeignKeyConstraint(
+            ["tenant_id"],
+            ["tenant.id"],
+        ),
+        sa.PrimaryKeyConstraint("id", "tenant_id"),
+    )
+    op.create_table(
+        "topologyserviceapplication",
+        sa.Column("tenant_id", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("service_id", sa.Integer(), nullable=False),
+        sa.Column("application_id", sa.Uuid(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["application_id", "tenant_id"],
+            ["topologyapplication.id", "topologyapplication.tenant_id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["service_id", "tenant_id"],
+            ["topologyservice.id", "topologyservice.tenant_id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["tenant_id"],
+            ["tenant.id"],
+        ),
+        sa.PrimaryKeyConstraint("tenant_id", "service_id", "application_id"),
+    )
+    op.create_table(
+        "topologyservicedependency",
+        sa.Column("id", sa.Uuid(), nullable=False),
+        sa.Column("tenant_id", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("service_id", sa.Integer(), nullable=False),
+        sa.Column("depends_on_service_id", sa.Integer(), nullable=False),
+        sa.Column("protocol", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.func.current_timestamp(),
+            nullable=True,
+        ),
+        sa.ForeignKeyConstraint(
+            ["depends_on_service_id", "tenant_id"],
+            ["topologyservice.id", "topologyservice.tenant_id"],
+            name="topologyservicedependency_depends_on_service_id_tenant_id_fkey",
+        ),
+        sa.ForeignKeyConstraint(
+            ["service_id", "tenant_id"],
+            ["topologyservice.id", "topologyservice.tenant_id"],
+            name="topologyservicedependency_service_id_tenant_id_fkey",
+        ),
+        sa.ForeignKeyConstraint(
+            ["tenant_id"],
+            ["tenant.id"],
+        ),
+        sa.PrimaryKeyConstraint("id", "tenant_id"),
+    )
+
+    transfer_data()
+
+    op.drop_table("topologyservicedependency_tmp")
+    op.drop_table("topologyserviceapplication_tmp")
+    op.drop_table("topologyapplication_tmp")
+    op.drop_table("topologyservice_tmp")
```

Comment:
May we not to drop it in this migration? I prefer us to keep data backup in the DB for a few versions & introduce a migration dropping data in a month - 2 when we're sure nothing is broken :)

Reply:
Done =)